ProjectLighthouse.Servers.Website/Pages/SlotPage.cshtml

```diff
@@ -26,12 +26,40 @@
 @if (Model.Slot!.Hidden)
 {
     <div class="ui inverted red segment">
-        <h2>This level is currently hidden.</h2>
-        <p><b>Only you and moderators may view this level.</b></p>
-        
-        <b>Reason:</b> <span>"@Model.Slot.HiddenReason"</span>
-        
-        <p><b>For more information please contact a moderator.</b></p>
+        <h3 style="margin-bottom:5px;"><i class="ban icon"></i> This level is currently hidden.</h3>
```

Comment:
Is the inline style necessary on the header? Could you move the icon instead or does that look weird?

Reply:
Yeah, give me a moment.